FrontEnd/src/components/ProfilePage/FormComponents/DeleteProfileComponent/DeleteProfileModal.js

```diff
@@ -42,22 +54,22 @@ const DeleteProfileModal = (props) => {
 
     const passwordVisisbilityHandler = () => {
         if (typePassword === 'password') {
-          setTypePassword('text');
+            setTypePassword('text');
         } else setTypePassword('password');
-      };
+    };
 
     return (
         <div>
             <div className={css['delete-header']}>Ви впевнені, що хочете видалити профіль?
-            <img
-            src={`${process.env.PUBLIC_URL}/profilepage/Vectorcancel.png`}
-            className={css['delete__cancelButton']}
-            alt=""
-            onClick={props.onCancel}/>
+                <img
+                    src={`${process.env.PUBLIC_URL}/profilepage/Vectorcancel.png`}
+                    className={css['delete__cancelButton']}
+                    alt=""
+                    onClick={props.onCancel} />
             </div>
             <div >
-                <div className={css['delete__description']}>Текст, який описує, що профіль буде видалено.</div>
-                <div className={css['delete__description']}>Також можуть бути перераховані способи відновити профіль.</div>
+                <div className={css['delete__description']}>Цей профіль буде видалено.
+                    Для того, щоб підтвердити видалення, будь-ласка, введіть вашу почту та пароль</div>
```

Comment:
Do we validate somewhere that the user inputs the correct data?

Reply:
We validate the email , but the password will be checked on the server.